Makefile

```diff
@@ -122,35 +124,65 @@ out/cypher-shell.zip: tmp/cypher-shell.zip
 	mkdir -p out
 	cp $< $@
 
-out/rpm/SPECS/cypher-shell.spec: packaging/rpm/cypher-shell.spec
+out/rpm8/SPECS/cypher-shell.spec: packaging/rpm8/cypher-shell.spec
 	mkdir -p $(dir $@)
 	VERSION=$(versionnumber) RELEASE=$(release) envsubst '$${VERSION} $${RELEASE}' < $< > $@
 
-out/rpm/BUILD/%: %
+out/rpm8/BUILD/%: %
 	mkdir -p $(dir $@)
 	cp $< $@
 
-out/%.rpm: out/rpm/RPMS/noarch/%.rpm
+out/%.rpm: out/rpm8/RPMS/noarch/%.rpm
 	mkdir -p $(dir $@)
 	cp $< $@
 
-tmp/rpm-test/%.rpm: out/rpm/RPMS/noarch/%.rpm
+tmp/rpm8-test/%.rpm: out/rpm8/RPMS/noarch/%.rpm
 	mkdir -p $(dir $@)
 	cp $< $@
 
-tmp/rpm-test/Dockerfile: packaging/test/rpm/Dockerfile
+tmp/rpm8-test/Dockerfile: packaging/test/rpm/Dockerfile
 	mkdir -p $(dir $@)
-	RPMFILE=$(rpmfile) envsubst '$${RPMFILE}' < $< > $@
+	RPMFILE=$(rpm8file) envsubst '$${RPMFILE}' < $< > $@
 
-out/rpm/RPMS/noarch/$(rpmfile): out/rpm/SPECS/cypher-shell.spec $(rpm_artifacts) out/rpm/BUILD/Makefile out/rpm/BUILD/cypher-shell.1.md
-	rpmbuild --define "_topdir $(CURDIR)/out/rpm" -bb --clean $<
+out/rpm8/RPMS/noarch/$(rpm8file): out/rpm8/SPECS/cypher-shell.spec $(rpm_artifacts) out/rpm8/BUILD/Makefile out/rpm8/BUILD/cypher-shell.1.md
+	rpmbuild --define "_topdir $(CURDIR)/out/rpm8" -bb --clean $<
 
-.PHONY: rpm
```

Comment:
There's now no longer a make target for rpm, so you can't do `make rpm` any more.

add a line somewhere:
`rpm: rpm8 rpm11`